server.py

```diff
@@ -1,238 +1,37 @@
 import asyncio
-import logging
-import uuid
-from ftplib import FTP, all_errors, error_perm
-from pathlib import Path
-from typing import List, Optional, Dict, Any
 from fastmcp import FastMCP
 from fastmcp.exceptions import ToolError
 
-# Create a custom logger
-LOGGER_NAME = "MCP_FTP_Client"
-logger = logging.getLogger(LOGGER_NAME)
-logger.setLevel(logging.INFO)
-
-# Create a formatter for the log messages
-formatter = logging.Formatter("[%(asctime)s] %(name)s: %(message)s", datefmt="%Y-%m-%d %H:%M:%S")
-
-# Create a handler to write logs to a file
-LOG_FILE_PATH = Path("mcp_ftp_client.log")
-file_handler = logging.FileHandler(LOG_FILE_PATH.absolute())
-file_handler.setFormatter(formatter)
-logger.addHandler(file_handler)
-
-# Create a handler to print logs to the console
-stream_handler = logging.StreamHandler()
-stream_handler.setFormatter(formatter)
-logger.addHandler(stream_handler)
+from ftp_client_logic import (
+    ftp_connect as ftp_connect_logic,
+    ftp_disconnect as ftp_disconnect_logic,
+    ftp_list as ftp_list_logic,
+    ftp_nlst as ftp_nlst_logic,
+    ftp_mlsd as ftp_mlsd_logic,
+    ftp_get as ftp_get_logic,
+    ftp_put as ftp_put_logic,
+    ftp_delete_recursive as ftp_delete_recursive_logic,
+    ftp_move as ftp_move_logic,
+    ftp_copy_recursive as ftp_copy_recursive_logic,
+    ftp_cd as ftp_cd_logic,
+    logger,
+)
```

Comment:
```suggestion
from ftp_client_logic import *
```
or
```suggestion
from ftp_client_logic import (ftp_connect, ftp_disconnect, ftp_list, ftp_nlst, ftp_mlsd, ftp_get, ftp_put, 
                                               ftp_delete_recursive, ftp_move, ftp_copy_recursive, ftp_cd, logger)
```

בעיניי אין צורך לשנות את השמות ב-import